src/snoowrap.js

```diff
@@ -238,6 +238,87 @@ const snoowrap = class snoowrap {
     });
   }
 
+  static get grantType () {
+    return {
+      CLIENT_CREDENTIALS: 'client_credentials',
+      INSTALLED_CLIENT: 'https://oauth.reddit.com/grants/installed_client'
+    };
+  }
+  /**
+  * @summary Creates a snoowrap requester from a "user-less" Authorization token
+  * @desc In some cases, 3rd party app clients may wish to make API requests without a user context. App clients can request
+  * a "user-less" Authorization token via either the standard client_credentials grant, or the reddit specific
+  * extension to this grant, https://oauth.reddit.com/grants/installed_client. Which grant type an app uses depends on
+  * the app-type and its use case.
+  * @param {object} options
+  * @param {string} options.userAgent A unique description of what your app does. This argument is not necessary when snoowrap
+  is running in a browser.
+  * @param {string} options.clientId The client ID of your app (assigned by reddit). If your code is running clientside in a
+  * browser, using an "Installed" app type is recommended. From the reddit docs: "reddit *may* choose to use this ID to generate
+  * aggregate data about user counts. Clients that wish to remain anonymous should use the value DO_NOT_TRACK_THIS_DEVICE."
+  * @param {string} [options.clientSecret] The client secret of your app. Only required for "Client" grant type.
```

Comment:
`"Client" grant type` here means CLIENT_CREDENTIALS, right? Should we make this clear since "client" also exists in "installed client"

Reply:
Done, should be much clearer now.